packages/lexical-devtools/src/panel/components/InspectedElementView/index.tsx

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+import './index.css';
+
+import {NodeProperties} from 'packages/lexical-devtools/types';
+import * as React from 'react';
+
+import InspectedElementProp from '../InspectedElementProp';
+
+function InspectedElementView({
+  nodeProps,
+}: {
+  nodeProps: NodeProperties | null;
+}): JSX.Element {
+  const propsDisplay = [];
+
+  if (nodeProps) {
+    for (const propName in nodeProps) {
+      propsDisplay.push(
+        <InspectedElementProp
+          propName={propName}
+          property={nodeProps[propName]}
+        />,
+      );
+    }
+  }
+
+  return <div className="inspected-element-view">{propsDisplay}</div>;
```

Comment:
There's probably a more concise way of just writing this logic: 
```suggestion
  nodeKeys: NodeProperties = {};
}): JSX.Element {
  return (
    <div className="inspected-element-view">{
     {Object.entries(rows).map(([key, value]) => (
        <InspectedElementProp
          propName={key}
          property={value}
        />
      ))}
     </div>
   );
```